src/main/kotlin/lotto/Lottos.kt

```diff
@@ -0,0 +1,40 @@
+package lotto
+
+import lotto.vo.Money
+
+class Lottos(
+    private val lottos: List<Lotto>,
+) : Iterable<Lotto> {
+    private val totalPrice = Money(LottoShop.LOTTO_PRICE.amount * lottos.count())
+
+    val size: Int = lottos.size
+
+    fun playWith(winningNumbers: List<Int>): GameResult {
+        val prizes = aggregateWinningPrizeWith(winningNumbers)
+
+        return GameResult(
+            prizes = prizes,
+            paidPrice = totalPrice
```

Comment:
trailing comma를 습관화 해주시면 좋습니다~!

https://kotlinlang.org/docs/coding-conventions.html#trailing-commas

```suggestion
            paidPrice = totalPrice,
```